### ISSUE TITLE
Initial conversion is broken for currency:null from 1.18.1 onwards

### ISSUE THREAD

**Issue by alesvaupotic** [dm4t2/vue-currency-input#118]
I based my Vuetify components on your example at [Codesandbox](https://codesandbox.io/s/using-vue-currency-input-with-vuetify-kd7d1) and since I would like my input to have always visible currency symbol, I added prefix or suffix on the input field and tried to disable it in component rendering.  It turns out it incorrectly handles the conversion with decimals in this case. I narrowed it down to 1.18.0 working as expected and described in the documentation

> A ISO 4217 currency code (for example USD or EUR). Default is EUR. You can also pass an object {prefix, suffix} for customizing the currency symbol or null to hide the currency symbol permanently.

Example at https://codesandbox.io/s/using-vue-currency-input-with-vuetify-bwdqq?file=/src/components/VCurrencyField.vue

![image](https://user-images.githubusercontent.com/1660347/82727473-13b38080-9ceb-11ea-96f7-04af3f792f32.png)

![image](https://user-images.githubusercontent.com/1660347/82727488-1f06ac00-9ceb-11ea-88c7-cdbe9363ee83.png)

If I missed something in the docs, please pardon my ignorance and just close this.

**Comment by dm4t2** [comment]
Hi, thanks for reporting. This is related to the changes introduced with #102. This affects only locales using a `.` as grouping symbol.

I just noticed a new API method for the initial formatting is needed to make it work fully again. In the meantime I added a work around until I will fix this in a new release:

https://codesandbox.io/s/vue-currency-input-118-ws9jf?file=/src/components/VCurrencyField.vue
